db/migrate/20210212120340_remove_position_index_from_content_page.rb

```diff
@@ -1,9 +1,15 @@
 class RemovePositionIndexFromContentPage < ActiveRecord::Migration[6.1]
   def up
```

Comment:
Can you create a new migration to remove the columns rather than change an existing migration.

Reply:
My mistake, I meant to do that and must have edited the wrong file

Crikey

Reply:
Done now